python/lsst/afw/fits/__init__.py

```diff
@@ -1,2 +1,2 @@
 from .fitsLib import FitsError, FitsTypeError, MemFileManager
-from .pickleFits import reduceToFits, unreduceFromFits
+#from .pickleFits import reduceToFits, unreduceFromFits
```

Comment:
pickleFits is a library that is not yet needed, and might not be needed at all. When we have finished wrapping afw, if this module (which currently breaks when we attempt to import it) is not needed, we can delete the line. Until then it should remain commented out.